### PR TITLE
Fix switching to manual address when adding bank account

### PR DESCRIPTION
**Pull request by francoisl**
Pullerbear review (@cead22)

### Details
When adding a bank account, fix the `Can't find your address? Enter it manually` link.

### Fixed Issues
$ https://github.com/Expensify/App/issues/6323

### Tests / QA
1. Create a workspace
2. Select the "Connect Bank Account"
3. Fill in the details until the step 3 "Personal Information"
4. Start typing an address in the "Personal Address" input, select an option in the dropdown below
5. Click "Can't find your address?"
5. Make sure the UI refreshes with the inputs to manually enter your address
6. Make sure the City, State, and Zip Code inputs are prepopulated with the values from the address you just selected previously

https://user-images.githubusercontent.com/2229301/142062764-d17802cb-3006-4c6e-bff7-99647ce574b2.mov

### Tested On

- [x] Web
- [ ] Mobile Web
- [ ] Desktop
- [ ] iOS
- [ ] Android

### CODE REVIEW

**Comment by Jag96** [comment]
Adding CP Staging label since this fixes a deploy blocker

**Comment by francoisl** [comment]
Ah yes thanks, I forgot.

File: src/pages/ReimbursementAccount/IdentityForm.js [hunk 1]
```diff
@@ -133,7 +133,7 @@ const IdentityForm = (props) => {
                 errorText={props.errors.ssnLast4 ? props.translate('bankAccount.error.ssnLast4') : ''}
                 maxLength={CONST.BANK_ACCOUNT.MAX_LENGTH.SSN}
             />
-            {props.manualAddress ? (
+            {props.values.manualAddress ? (
```
Comment (Jag96) [inline src/pages/ReimbursementAccount/IdentityForm.js:136]: Do the other props need to be updated as well? For example, `value={props.city}` is still used even though `city` is inside `values` in the propTypes definition.

Comment (francoisl) [reply src/pages/ReimbursementAccount/IdentityForm.js:136]: Uh yeah good thinking, probably. I was too focused on fixing the issue and didn't think to check the rest,  I'll check now.

**Comment by francoisl** [comment]
Updated code and test steps.

**Review by Jag96 — APPROVED** [review]
Looks good, confirmed it works on mobile as well 🚀

**Pull request merged**